carveracontroller/main.py

```diff
@@ -92,7 +92,11 @@ def switch_lang(self, lang):
 from kivy.config import Config
 from kivy.metrics import Metrics
 
-from serial.tools.list_ports import comports
+try:
+    from serial.tools.list_ports import comports
+except:
```

Comment:
bare except isn't great. I think this is the right one.
```suggestion
except ImportError:
```

Reply:
SUre